### ISSUE TITLE
recreate all the PNG files

### ISSUE THREAD

**Issue by mbjones** [NCEAS/eml#223]
---

Author Name: **Margaret O'Brien** (Margaret O'Brien)
Original Redmine Issue: 3600, https://projects.ecoinformatics.org/ecoinfo/issues/3600
Original Date: 2008-11-06
Original Assignee: Matt Jones

---

Two 2.1 upgrades in particular:
1. method ->methods, and datetime ->dateTime
2. retyping geographicCoverage elements
affected almost all the schemas. The best solution is to recreate all the PNG files with xmlspy. very tedious, sorry. Use the existing online images as templates to be sure that expansion of complex types is consistent.

changes to "methods" affects:
dataTable
entity
spatialRaster
spatialVector
storedProcedure
view

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Margaret O'Brien** (Margaret O'Brien)
Original Date: 2008-12-03T23:21:33Z

---

These pngs need to be replaced:
coverage
dataset
dataTable
entity
eml
literature
physical
protocol
software
spatialRaster
spatialReference
spatialVector
storedProcedure
text
view

but not these (no changes affected the pngs):
access
attribute
constraint
methods
party
project
protocol

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Margaret O'Brien** (Margaret O'Brien)
Original Date: 2008-12-04T20:07:08Z

---

These pngs were replaced:
coverage, dataset, dataTable, entity, eml, literature, physical, protocol, esource, software, spatialRaster, spatialVector, storedProcedure, text, view

not affected by schema changes:
access, attribute, constraint, methods, party, project, spatialReference

For the most part, nodes were expanded to the same level as in the EML201 documentation. However, I tried to standardize the expansions a bit more: I expanded the types in the schema diagram where they were defined and collapsed them where they were imported. For example, this meant replacing the resource.png so that the changes to the distributionType could be illustrated. This tree had not been expanded in the 201 png, and so the existing diagram was accurate as is. But there is no other diagram where the res:DistributionType could be guaranteed to be expanded, and it should be, since it is different from phys:PhysicalDistributionType.
